### DISCUSSION TITLE
Delete Tables through GUI

### DISCUSSION THREAD

**Discussion by kr886q** [beekeeper-studio/beekeeper-studio#973] [Ideas]
I know I can just run a query to DROP TABLE table_name ; but it would be nice to right click and drop a table from the gui

**Comment by mvuidev** [comment]
+1

Just started to use Beekeeper, seems nice, but maybe missing some GUI actions like "remove table" or even "create database" ?

**Comment by rathboma** [comment]
Coming soon :-) PRs are already in
